spec/system/activities_spec.rb

```diff
@@ -233,6 +233,77 @@
     end
   end
 
+  context 'as a group admin' do
+    let!(:group_admin)    { create(:group_admin, school: school)}
```

Comment:
This is setting up a group admin that is associated with both a school group and a school. I don't think that will happen in practice, so the tests might well not be testing the revised behaviour especially around rights to record activities?